plugin/audit_log/tests/mtr/audit_log_filter_db_events.inc

```diff
@@ -0,0 +1,30 @@
+
+
+CREATE TABLE db1.t (a INT);
```

Comment:
I'd add some more tests (strictly speaking to test MYSQL_AUDIT_TABLE_ACCESS_CLASS notifications and not this plugin, but if we ship this we have to have confidence if the whole thing works):
- multi-table delete with same table listed twice
- trigger on a table that writes to a different table
- table access through event scheduler (actually this one is interesting to test outside the db filtering context too)